### ISSUE TITLE
Deleting database.yml causes failure to boot

### ISSUE THREAD

**Issue by adam-harwood** [tongueroo/jets#78]
My project doesn't use any SQL database, so I removed the mysql2 Gem dependency and the database.yml file. When I run jets server (or deploy), I receive the error message below. I assume this is because some part of activerecord or activesupport is needed, but isn't loaded because of the lack of DB connection. I suspect I can come up with some workaround for this, but any idea what the proper fix might be? Happy to work on a pull request if I have the time, but I don't really know where to start on it.

```
Boot Error

Something went wrong while loading config.ru
NoMethodError: undefined method `class_attribute' for Jets::Turbine:Class Did you mean? alias_attribute

/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/jets-1.1.2/lib/jets/turbine.rb:3:in `<class:Turbine>'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/jets-1.1.2/lib/jets/turbine.rb:2:in `<module:Jets>'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/jets-1.1.2/lib/jets/turbine.rb:1:in `<top (required)>'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `require'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `block in require'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:253:in `load_dependency'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `require'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/jets-1.1.2/lib/jets/booter.rb:25:in `turbine_initializers'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/jets-1.1.2/lib/jets/booter.rb:13:in `boot!'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/jets-1.1.2/lib/jets/core.rb:31:in `boot'
config.ru:4:in `block in inner_app'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rack-2.0.6/lib/rack/builder.rb:55:in `instance_eval'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rack-2.0.6/lib/rack/builder.rb:55:in `initialize'
config.ru:1:in `new'
config.ru:1:in `inner_app'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/shotgun-0.9.2/lib/shotgun/loader.rb:113:in `eval'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/shotgun-0.9.2/lib/shotgun/loader.rb:113:in `inner_app'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/shotgun-0.9.2/lib/shotgun/loader.rb:103:in `assemble_app'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/shotgun-0.9.2/lib/shotgun/loader.rb:86:in `proceed_as_child'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/shotgun-0.9.2/lib/shotgun/loader.rb:31:in `call!'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/shotgun-0.9.2/lib/shotgun/loader.rb:18:in `call'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/shotgun-0.9.2/lib/shotgun/favicon.rb:12:in `call'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/shotgun-0.9.2/lib/shotgun/static.rb:14:in `call'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rack-2.0.6/lib/rack/urlmap.rb:68:in `block in call'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rack-2.0.6/lib/rack/urlmap.rb:53:in `each'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rack-2.0.6/lib/rack/urlmap.rb:53:in `call'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rack-2.0.6/lib/rack/builder.rb:153:in `call'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/rack-2.0.6/lib/rack/handler/webrick.rb:86:in `service'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/2.5.0/webrick/httpserver.rb:140:in `service'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/2.5.0/webrick/httpserver.rb:96:in `run'
/home/aharwood/.rbenv/versions/2.5.1/lib/ruby/2.5.0/webrick/server.rb:307:in `block in start_thread'
```

**Comment by tongueroo** [comment]
@adam-harwood Thanks for the report. Just had to require activesupport correctly.

Fixed in #79 Released in v1.1.3 [CHANGELOG](https://github.com/tongueroo/jets/blob/master/CHANGELOG.md)

Tested with:

    jets new demo --no-database
    cd demo
    jets server

Then localhost:8888 no longer throws that error.

**Comment by adam-harwood** [comment]
Confirmed working, thank you!
